situp-plugins/service-map-stateful/src/main/java/com/amazon/situp/plugins/processor/ServiceMapStatefulProcessor.java

```diff
@@ -225,10 +226,10 @@ private String getTraceGroupName(final byte[] traceId) {
      * Delete current state held in the processor
      */
     public void deleteState() {
```

Comment:
This is not used, can we remove this?

Reply:
I was using it in the test class, but I can just manually clean the directory